src/Common/Connectivity/Dependencies.generator.js

```diff
@@ -6,7 +6,7 @@ const _ = require("lodash")
 const dependenciesTemplate = Handlebars.compile(fs.readFileSync("src/Common/Connectivity/DependencyGraph.template.hbs","utf8"))
 const connectorTemplate = Handlebars.compile(fs.readFileSync("src/Common/Connectivity/Connector.template.hbs","utf8"))
 
-let dependencies = yaml.safeLoad(fs.readFileSync("src/Common/Connectivity/Dependencies.yml","utf8"));
+let dependencies = yaml.load(fs.readFileSync("src/Common/Connectivity/Dependencies.yml","utf8"));
```

Comment:
Appears to rely on a recent `js-yaml` but the version is not explicitly stated within package.json?

Reply:
Still not sure about this one. I'll be pushing a new commit to take care of some of the feedback from this PR in a moment.

What is the problem with safeLoad? Noting that the functionality is the same in js-yaml@4, but js-yaml@3 appears to be installed. Due to #40, right now I'm thinking to leave this like it is.